src/heretic/mn_menu.c

```diff
@@ -661,7 +681,8 @@ void MN_Drawer(void)
         {
             if (item->type != ITT_EMPTY && item->text)
             {
-                if (CurrentMenu == &CrispnessMenu)
+                if (CurrentMenu == &Crispness1Menu ||
```

Comment:
You could either macro'ize it. Or, in Doom, I have introduced a pointer that always points to the current Crispness menu page.

Reply:
I redid the Crispness menu code and this check is much simpler now.